<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background: rgb( 128, 128, 128);
    /*
	border-style: outset;
    border-width: 2px;
    border-radius: 4px;
    border-color: beige;
	//*/
}

QLabel{
	color: rgb(255, 255, 255);
}

QSpinBox{
	color: rgb(255, 255, 255);
	background: rgb( 128, 128, 128);
	border-width: 0;
}

QDoubleSpinBox{
	color: rgb(255, 255, 255);
	background: rgb( 128, 128, 128);
}

QPushButton {
     background-color: rgb( 196, 196, 196);
     border-style: outset;
     border-width: 1px;
     border-radius: 2px;
     border-color: beige;
     min-width: 5em;
 }

 QPushButton:pressed {
     background-color: rgb( 196, 196, 196);
     border-style: inset;
 }

QCheckBox {
	color: rgb(255, 255, 255);
	background: rgb( 128, 128, 128);
 }

QCheckBox {
     spacing: 5px;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
	 color: rgb(255, 255, 255);
	 background: rgb( 128, 128, 128);
     border-style: outset;
     border-width: 1px;
     border-radius: 2px;
     border-color: beige;
 }

 QCheckBox::indicator:unchecked {
	background: rgb( 128, 128, 128);
 }

 QCheckBox::indicator:unchecked:hover {
	background: rgb( 169, 169, 169);
 }

 QCheckBox::indicator:unchecked:pressed {
	background: rgb( 200, 200, 200);
 }

 QCheckBox::indicator:checked {
	background: rgb(92, 255, 0);
 }

 QCheckBox::indicator:checked:hover {
     background: rgb(0, 255, 0);
 }

 QCheckBox::indicator:checked:pressed {
     background: rgb(0, 255, 0);
 }

 QCheckBox::indicator:indeterminate:hover {
     background: transparent;
 }

 QCheckBox::indicator:indeterminate:pressed {
     background: transparent;
 }</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Countdown time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QSpinBox" name="hrsSpin">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minSpin">
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="secSpin">
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Window transparency:</string>
       </property>
       <property name="buddy">
        <cstring>opacitySpin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="opacitySpin">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.800000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Show dialog when finished:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="showDialogBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Show seconds:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="showSecBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>lgTimer 1.1 by lightgraffiti.de published in the public domain</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Icons are provided by GLYPHICONS.com und the Creative Commons Attribution 3.0 Unported (CC BY 3.0)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
